zio-http/src/main/scala/zio/http/codec/HttpCodec.scala

```diff
@@ -273,6 +274,14 @@ object HttpCodec
     with StatusCodecs {
   implicit def stringToLiteral(string: String): PathCodec[Unit] = literal(string)
 
+  def enumeration[AtomType, Value]: Enumeration[AtomType, Value] =
```

Comment:
Due to contravariance of `AtomType`, can we just let it widen automatically?

Reply:
That would reduce it down to a single type parameter, unless there's some problem with that approach.